_allevents/reimagine-storytelling-pitch-ct.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: default 
+title: "Re-imagine Storytelling Open Pitch Session: Cape Town"
+description: Join us for the Open Pitch Session of Re-Imagine Storytelling, a challenge that combines elements of activism, journalism, data, and tech.
+image: /images/launch-ct.jpg
+ev-date: 2016-09-06
+type: none
+link: /allevents/reimagine-storytelling-pitch-ct.html
+location: Cape Town
+---
+<div class="container contact">
+	<div class="row">
+		<div class="col-sm-12">
+			<h2>Re-imagine Storytelling Open Pitch Session</h2>
+		</div>
+	</div>
+
+	<div class="row">
+		<div class="col-sm-6">
+			<h4 style="margin-top:0"><span class="label label-date">06 September 2016</span> <span class="label label-location">Cape Town</span></h4>
+			<p>Join us for the Open Pitch Session of Re-Imagine Storytelling, a challenge that combines elements of
+			<ul class="fa-ul">
+				<li><i class="fa fa-child fa-fw"></i> Activism</li>
+				<li><i class="fa fa-pencil fa-fw"></i> Journalism</li>
+				<li><i class="fa fa-database fa-fw"></i> Data</li>
+				<li><i class="fa fa-code fa-fw"></i> Tech</li>
+			</ul>
+			Anyone interested in entering the Storytelling challenge, as well as any organisations who want to take part, are encouraged to join us. Read our <a href="/2016-08-25/reimagine-storytelling.htm">blog post</a> for more information about the challenge. 
+			</p>
+			<div class="well">
+				<p>
+					<i class="fa fa-map-marker fa-fw"></i> <a href="/contact">Codebridge, Claremont, Cape Town</a><br>
+					<i class="fa fa-calendar-o fa-fw"></i> Tuesday, 6 September 2016<br>
+					<i class="fa fa-clock-o fa-fw"></i> 18:30 for 19:00
+				</p>
+				<a class="btn btn-default" href="https://docs.google.com/a/code4sa.org/forms/d/15GPUWN54VtBM6pBeteqIPO3zDG4XnmNDu2pZ5NidqXE/edit" target="_blank"><strong>RSVP</strong></a>
```

Comment:
This links to the edit version of the form...not the version people can use to complete